src/components/NewsLink.jsx

```diff
@@ -0,0 +1,17 @@
+import { DateNewsCard } from "@baltimorecounty/dotgov-components";
+import React from "react";
+
+const NewsLink = ({ ArticleSummary, Article, url }) => {
+  return (
+    <React.Fragment>
+      <DateNewsCard
+        date="07/29/2019"
```

Comment:
also your date and description should come from props.

Reply:
this control is actually not needed and not used. I thought I had deleted it